brew/status.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+set -u
+
+eval $(brew/vars.sh)
+
+echo file checks \(exes, state, socket\)...
+ls -la $TS_BIN/tailscale*
+ls -l $TS_STATE
+ls -l $TS_SOCK
+
+echo
+$TS_BIN/tailscale version
+
+echo
+$TS_BIN/tailscale netcheck
+
+echo
+echo tailscale status:
+sudo $TS_BIN/tailscale -socket=$TS_SOCK status
+
+echo
+ifconfig -r en0
+
+# TODO(mkramlich): handle tun right everywhere below
```

Comment:
Not sure what the best way to handle this is. Maybe doing a diff of the `ifconfig` output?

Reply:
yeah agreed

not sure theres one right way to do this. might depend on the situation. honestly status.sh is the most ad hoc and subjective of the brew scripts. I planned to document better in the README and refine this. For a brew maintainer there are 2 distinct concerns: will my brew package candidate work right? does the tailscale build work "right" (or, as expected)? brew maintainer cares most about the former, but certain problems can blur the line. thus, most of status.sh is a smoke test report, and happened to emphasize areas I've wanted to peek into when investigating other issues, like routing. so, very arbitrary and very very WIP there. least polish. I tried to make that point in the script comments